asr1k_neutron_l3/models/neutron/l3/router.py

```diff
@@ -150,17 +150,22 @@ def get_internal_cidrs(self):
 
     def _build_routes(self):
         routes = route.RouteCollection(self.router_id)
-        primary_route = self._primary_route()
-        if primary_route is not None and self._route_has_connected_interface(primary_route):
-            routes.append(primary_route)
 
+        # In case the customer sets a default route, we will restrain from programming the openstack primary route
+        overwrite_primary = False
```

Comment:
I think I'd've named it `primary_overwritten`, as the customer decided to override the primary, not us. Also one could have [long discussions about if this should be overwrite or override](https://stackoverflow.com/questions/8651562/overwrite-or-override/8651622) (and I'd go with override), but it's nothing you need to change if you don't want to.

Reply:
Accepted.